loki-client/src/main/java/com/github/loki4j/client/pipeline/PipelineConfig.java

```diff
@@ -187,6 +196,17 @@ public static class Builder {
         private long sendQueueMaxBytes = batchMaxBytes * 10;
         private int maxRetries = 2;
         private long retryTimeoutMs = 60 * 1000;
+        private Supplier<Long> jitter = new Jitter();
+        private BiFunction<Integer, Long, Boolean> sleep = (attempt, timeout) -> {
+            try {
+                long backoff = timeout * (1 << (attempt - 1));
+                Thread.sleep(backoff + jitter.get());
```

Comment:
Let's explicitly specify we do our calculations in ms

Reply:
Done.